rules/windows/defense_evasion_masquerading_suspicious_werfault_childproc.toml

```
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/08/24"
+ecs_version = ["1.5.0"]
```

```suggestion
ecs_version = ["1.6.0"]
```

Can accept change once #220 merges